### PR TITLE
Bci repo

### PR DESCRIPTION
**Pull request by mloviska**
Obsolete and duplicate openQA BCI repo code with BCI repository test as it is
executed against fedora image (with `dnf`) in order to perform more
repoclosure check over BCI repository.

- Related ticket: https://progress.opensuse.org/issues/132935

### CODE REVIEW

**Comment by mloviska** [comment]
VRs: https://github.com/SUSE/BCI-tests/pull/274#issuecomment-1641905220
Job group: https://gitlab.suse.de/qac/qac-openqa-yaml/-/merge_requests/1275

File: tests/containers/bci_test.pm [hunk 1]
```diff
@@ -99,6 +99,9 @@ sub run {
     my $engine = $args->{runtime};
     my $bci_devel_repo = get_var('BCI_DEVEL_REPO');
     my $bci_tests_repo = get_required_var('BCI_TESTS_REPO');
+    if (my $bci_repo = get_var('REPO_BCI')) {
+        $bci_tests_repo = "http://openqa.suse.de/assets/repo/$bci_repo";
```
Comment (grisu48) [inline tests/containers/bci_test.pm:103]: Judging from the test module, this variable is actually never being used. I see the VRs are passing but are we sure, that they are using the desired BCI repo? :thinking:

Comment (mloviska) [reply tests/containers/bci_test.pm]: Right, good catch! Thanks

http://kepler.suse.cz/tests/21329#step/bci_test_podman/40

File: lib/main_containers.pm [hunk 2]
```diff
@@ -243,9 +243,6 @@ sub load_container_tests {
                 load_image_tests_in_k8s($run_args) if (/k8s/i);
                 load_image_tests_in_openshift if (/openshift/i);
             }
-        } elsif (get_var('REPO_BCI')) {
-            loadtest 'containers/host_configuration';
-            loadtest 'containers/bci_repo';
```
Comment (jlausuch) [inline lib/main_containers.pm:248]: wait, if you remove this, how is these jobs now scheduled?

Comment (mloviska) [reply lib/main_containers.pm:248]: It would be scheduled by ibs sync and I have put the related variables into job group.

Comment (jlausuch) [reply lib/main_containers.pm:248]: Ah ok, it's scheduled because is_container_image_test() returns 1 due to `BCI_TESTS=1`.

**Review by jlausuch — APPROVED** [review]

**Pull request merged**